test/NuGet.Tests.Apex/NuGet.Console.TestContract/ApexTestConsole.cs

```diff
@@ -67,8 +67,8 @@ public bool ConsoleContainsMessage(string message)
             {
                 var snapshotLine = snapshot.GetLineFromLineNumber(i);
                 var lineText = snapshotLine.GetText();
-
-                var foundMessage = lineText.Contains(message);
+                var foundMessage = lineText.IndexOf(message, StringComparison.OrdinalIgnoreCase) >= 0;
```

Comment:
Why change `Contains` to `IndexOf`?

Reply:
`Contains` calls `IndexOf` so that on it's own won't do much...can you do ignore case in contains?

Reply:
there is no way to do ignore case comparison via contains.